packages/agent/src/oidc.ts

```diff
@@ -600,18 +594,116 @@ function encryptAuthResponse({
   return compactJwe;
 }
 
+export async function createPermissionGrants(
+  selectedDids: string[],
+  ephemeralDid: BearerDid,
```

Comment:
Sanity: did you mean app DID or delegated DID?

Reply:
another good catch. changed